go.mod

```diff
@@ -38,6 +39,7 @@ require (
 	cel.dev/expr v0.24.0 // indirect
 	cloud.google.com/go/auth v0.16.3 // indirect
 	cloud.google.com/go/auth/oauth2adapt v0.2.8 // indirect
+	github.com/StackExchange/wmi v1.2.1 // indirect
```

Comment:
this is also an archived repo, last updated four years ago

Reply:
Yeah, unfortunately there just isn't a great alternative to this for Windows Management Interface support (which is what we rely on in ghw for Windows inspection support)